doc/src/manual/modules.md

```diff
@@ -322,6 +322,68 @@ Here, Julia cannot decide which `f` you are referring to, so you have to make a
 
 3. When the names in question *do* share a meaning, it is common for one module to import it from another, or have a lightweight “base” package with the sole function of defining an interface like this, which can be used by other packages. It is conventional to have such package names end in `...Base` (which has nothing to do with Julia's `Base` module).
 
+### Precedence order of definitions
+
+There are in general four kinds sources of binding definitions:
+   1. Those provided via implicit import through `using M`
+   2. Those provided via explicit import (e.g. `using M: x`)
+   3. Those declared as global scope `global x`
+   4. Those declared using definition syntax (`const`, `global x::T`, `struct`, `etc`)
+
+Syntactically, we divide these into three precedence levels (from weakest to strongest)
+   1. Implicit imports
+   2. Implicit declarations
+   3. Explicit declarations and imports
```

Comment:
the terms "implicit import" (1) and "explicit import" (2) are included in the list of 4 kinds above, but the terms "implicit declarations" and "explicit declarations" are not -- should they be? Is `global x` (3) implicit declaration , but `const x = 1` or `global x::T` (4) explicit declaration?

Reply:
Yes, I'll tweak the text